test/controllers/hosts_controller_test.rb

```diff
@@ -828,7 +828,7 @@ def setup_multiple_compute_resource
 
     test "REMOTE_USER should be trusted for UI requests" do
       get :show, params: {:id => @host.to_param}
-      assert_response :success
+      assert_redirected_to "/users/extlogin"
```

Comment:
This isn't a correct change, the `setup` block should set up an admin user with external auth to check it works.
A similar change is required to get the api controller test to pass.